source/index.html.md

```diff
@@ -1406,22 +1408,758 @@ can still only capture 5.
+
+<p class="alert alert-info">
+<b>Notice:</b> Debit is supported only by Visa. Currently, only merchants with selected MCCs
+and assigned BAIs are able to process debits; namely exactly those that will
+result in the debit being an AFT.
+</p>
 
 #### Parameters
 
-````
-POST https://gateway.clearhaus.com/authorizations/:id/voids
-````
+<dl class="dl-vertical">
+  <dt>amount
+    <span class="type">(0|[1-9][0-9]{,8})</span>
+  </dt>
+  <dd>
+    Amount in minor units of given currency (e.g. cents if in Euro).
+  </dd>
+
+  <dt>currency
+    <span class="type">[A-Z]{3}</span>
+  </dt>
+  <dd>
+    <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some
+    exponents differ from ISO 4217.)
+  </dd>
+
+  <dt>credential_on_file
+    <span class="type">(<code>store</code>|<code>use</code>)</span>
+  </dt>
+  <dd>
+    Indicate if the payment credential (e.g. PAN and expiry) will be stored for
+    future use where the payment credential is not provided by the
+    cardholder but collected from (encrypted) storage.
+    <br />
+    <code>store</code>: The payment credential will be stored; it may only be
+    stored if the debit is approved.
+    <br />
+    <code>use</code>: The payment credential has already been stored and is now
+    being used.
+    <br />
+    Default:
+    <ul>
+      <li><code>use</code>, if <code>initiator</code> is <code>merchant</code>
+      (<code>store</code> is invalid),</li>
+      <li><code>store</code>, if the debit is first-in-series.</li>
+    </ul>
+
+    <div class="type">Optional</code>
+  </dd>
+
+  <dt>initiator
+    <span class="type">(<code>cardholder</code>|<code>merchant</code>)</span>
+  </dt>
+  <dd>
+    The initiator of the debit. A debit is initiated by the cardholder if the
+    cardholder decided the transaction should be created. This is regardless of
+    whether a stored payment credential is being used.
+    <br />
+    For compliance reasons there should be a previous approved transaction (for
+    the combination of card and merchant) where
+    <code>credential_on_file=store</code> before <code>initiator</code> may be
+    <code>merchant</code>.
+    <br />
+    Default:
+    <ul>
+      <li><code>merchant</code>, if the debit is
+      subsequent-in-series (<code>cardholder</code> is invalid),</li>
+      <li><code>cardholder</code>, otherwise.</li>
+    </ul>
+
+    <div class="type">Optional</div>
+  </dd>
+
+  <dt>ip
+    <span class="type">[0-9.a-fA-F:]{3,45}</span>
+  </dt>
+  <dd>
+    Cardholder's IP address. It must be a valid v4 or v6 address.
+    <div class="type">Optional<br></div>
+  </dd>
+
+  <dt>reference
+    <span class="type">[\x20-\x7E]{1,30}
+      <a target="_blank" href= "https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">
+        ASCII printable characters
+      </a>
+    </span>
+  </dt>
+  <dd>
+    A reference to an external object, such as an order number.
+    <div class="type">Optional</div>
+  </dd>
+
+  <dt>
+    series[type]
```

Comment:
Will the **debit** type support being in-series already? I was under the impression that this feature would be added at some later point.

Reply:
We'll see how it unfolds — if it's "sufficiently easy" to add it right away, we'll do that; if it ends up being a larger one we may cut the scope initially. However, you're right that it'll very likely come eventually.